ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -376,6 +364,27 @@ export default React.memo<Props>(function VideoJs(props: Props) {
         vjsPlayer = window.player;
       }
 
+      // hide unused elements on livestream
+      if (isLivestream) {
+        vjsPlayer.addClass('vjs-live');
+        vjsPlayer.addClass('vjs-liveui');
+        // $FlowIssue
```

Comment:
Continuing from the previous comment ... doubtful this will work reliably, but then again not sure how things work now.  Looks like it is circumventing the framework and assumes that this is all that is needed to reload things.

Looking at the their code, there is a bunch of listeners that need to be re/un-installed to really complete things.  If we just remove the ui but not the listener ... what if the listener-handler does something unexpected?
```
  /**
   * start/stop tracking
   */
  toggleTracking() {
    if (this.player_.duration() === Infinity && this.liveWindow() >= this.options_.trackingThreshold) {
      if (this.player_.options_.liveui) {
        this.player_.addClass('vjs-liveui');
      }
      this.startTracking();
    } else {
      this.player_.removeClass('vjs-liveui');
      this.stopTracking();
    }
  }

  /**
   * start tracking live playback
   */
  startTracking() {
    if (this.isTracking()) {
      return;
    }

    // If we haven't seen a timeupdate, we need to check whether playback
    // began before this component started tracking. This can happen commonly
    // when using autoplay.
    if (!this.timeupdateSeen_) {
      this.timeupdateSeen_ = this.player_.hasStarted();
    }

    this.trackingInterval_ = this.setInterval(this.trackLiveHandler_, Fn.UPDATE_REFRESH_INTERVAL);
    this.trackLive_();

    this.on(this.player_, ['play', 'pause'], this.trackLiveHandler_);

    if (!this.timeupdateSeen_) {
      this.one(this.player_, 'play', this.handlePlay_);
      this.one(this.player_, 'timeupdate', this.handleFirstTimeupdate_);
    } else {
      this.on(this.player_, 'seeked', this.handleSeeked_);
    }
  }
```